env0/resource_environment_test.go

```diff
@@ -2011,6 +2011,75 @@ func TestUnitEnvironmentResource(t *testing.T) {
 				mock.EXPECT().EnvironmentDestroy(environment.Id).Times(1)
 			})
 		})
+
+		t.Run("No drift with unspecified fields", func(t *testing.T) {
```

Comment:
There are 2 cases:
1. Environment can inherit the fields from the project. So given a `env0_project` and `env0_project_policy` that is specifying one of these values, the environment should not be created with a drift. 
This is similar to what you did, but I couldn't see where the `env0_project` and `env0_project_policy` are defined.
Can we test that? (might require some change to existing test)
2. `env0_environment` is defined with these fields, but then upon changing some value on the `env0_environment` resource, we should detect a drift for the environment resource... 
Can we have a test for that?

Reply:
we spoke, we'll go with this test for now. It gives some coverage by simulating the API result for the env0_environment entity and checks it doesn't create it with the inherited fields in the state